src/main/java/frc/team2412/robot/commands/index/IndexShootCommand.java

```diff
@@ -15,13 +15,16 @@ public IndexShootCommand(IndexSubsystem subsystem) {
     public void initialize() {
         // turn on both motors
         subsystem.ingestMotorIn();
-        subsystem.feederMotorIn();
+        subsystem.feederMotorInFullSpeed();
+        System.out.println("*****************SHOOTING NOW*********************");
```

Comment:
We will likely want to remove or make the shooter log spam configurable or disable completely before DCMP.